<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWidget</class>
 <widget class="QWidget" name="TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="aSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="bSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="cSlider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>aSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
